### DISCUSSION TITLE
Building Demo in flutter

### DISCUSSION THREAD

**Discussion by edunickdev** [deepgram/community#123] [General help] [answered]
Hi, i'm building a demo app to show it to my team, i'm following the next tutorial https://blog.deepgram.com/flutter-speech-to-text-tutorial/, but still with this issue in console, 
![image](https://user-images.githubusercontent.com/114629663/232147806-8edd2ae5-aedb-4d87-acb4-685611fd78db.png)

my configuration in the graddle file is: 
![image](https://user-images.githubusercontent.com/114629663/232149248-abb393b1-a31b-494f-91ee-382d7b6006b1.png)

i followed all the steps carefully, i´ll appreciate the help to solve this, if need more information only asked me.

Pd: got my api key ok and i´m using of endpoint this, https://api.deepgram.com/v1.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @edunickdev apologies for the late reply. We updated our Flutter guide + code on Monday which includes updates to the dependencies. Hopefully that will fix your issues. Here's the link: https://deepgram.com/learn/flutter-speech-to-text-tutorial
